common/sketch/src/main/java/org/apache/spark/util/sketch/BloomFilter.java

```diff
@@ -81,6 +81,11 @@ int getVersionNumber() {
   public abstract long bitSize();
 
   /**
+   * Swamidass & Baldi (2007) approximation for number of items in a Bloom filter
+   */
+  public abstract double approxItems();
```

Comment:
It might be easier to keep it as double because the estimate could be out of bound if the bits are full.